doc/setting-up-dex.md

```diff
@@ -0,0 +1,307 @@
+# Setting up Dex
+
+## What is Dex?
+
+[Dex][tool-dex] is an identity service that uses [OpenID Connect][oidc] to
+drive authentication for other apps.
+
+[tool-dex]: asd
+[oidc]: asd
+
+## Boostrap Flux
+
+TODO
```

Comment:
is this coming now or later?

Reply:
I think this file meant to be deleted as we did the documentation under `website/`

Reply:
Will remove the `Boostrap Flux` and add it as a pre-requirement